libs/elements/banners/src/lib/components/elewa-image-and-details-banner/elewa-image-and-details-banner.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+
+import { ImageAndDetails } from '@elewa-website/models/schema/ui/banners';
+
+@Component({
+  selector: 'elewa-image-and-details-banner',
+  templateUrl: './elewa-image-and-details-banner.component.html',
+  styleUrls: ['./elewa-image-and-details-banner.component.scss'],
+})
+export class ElewaImageAndDetailsBannerComponent {
+  @Input() imageAnddetails!: ImageAndDetails;
+
+    /**
+   * Get the CSS style for the image position based on the provided string.
+   *
+   * @param {string} imagePosition - The position of the image ('left' or 'right').
+   * @returns {{ [key: string]: string }} - An object representing CSS styles.
+   */
+
+  getPositionStyle(imagePosition: string) :string  {
+    switch (imagePosition) {
+      case 'left':
+        return "info-card-image-left";
+      case 'right':
+        return "info-card-image-right";
+      default:
+        return ""; // default value
+    }
```

Comment:
Since you're taking this approach your method can just return the imagePosition No need for the switch case. And in your CSS just work with left and right classnames. 

And since the method will now just be returning the imagePosition in your HTML you can have it as `[ngClass]="imageAnddetails.imagePosition`

Reply:
alright working on it.

Reply:
@LemmyMwaura I'm done